### PR TITLE
[10.0] Migrate 

### PR DESCRIPTION
**Pull request by osvalr**
## This change includes:

 - [x] Update travis conf
 - [x] Migrate `l10n_pe_toponyms` up to 10.0
 - [x] Migrate `opl` up to 10.0
 - [x] Migrate `opl_all` up to 10.0
 - [x] Migrate `l10n_pe_vat_sunat_validation` up to 10.0
 - [x] Set `sales_team` instead of `base` in `l10n_pe_toponyms` due users groups were moved in core addons from **`openerp/addons/base/res/res_country_view.xml`** to **`addons/sales_team/views/crm_team_views.xml`**

### CODE REVIEW

**Comment by osvalr** [comment]
@nhomar @luistorresm can you guys help me to review this, please?

File: l10n_pe_toponyms/__manifest__.py [hunk 1]
```diff
@@ -4,9 +4,12 @@
 
 {
     'name': 'l10n_pe Geopolitical Distribution',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Localization/Peru',
-    'depends': ['base'],
+    'depends': [
+        'base',
```
Comment (luistorresm) [inline l10n_pe_toponyms/__manifest__.py]: Could you remove base dependency, please?

File: l10n_pe_vat_sunat_validation/models/res_partner.py [hunk 2]
```diff
@@ -77,5 +74,8 @@ def onchange_vat(self):
                 district_code = 'PE%(code)s' % {'code': result.get('city', '')}
                 district_brw = district_obj.\
                     search([('code', '=', district_code)])
-                result.update({'l10n_pe_district_id': district_brw.id})
+                result.update({
+                    'l10n_pe_district_id': district_brw.id,
+                    'zip': district_brw.code[2:],
```
Comment (luistorresm) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: Why you add the new key 'zip'?

Comment (osvalr) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: hi @luistorresm, this zip code was [required](https://www.vauxoo.com/web#id=1364&view_type=form&model=user.story&menu_id=557&action=726) to be filled

File: l10n_pe_toponyms/__manifest__.py (hunk 1 above)
Comment (osvalr) [inline l10n_pe_toponyms/__manifest__.py]: sure, thanks

**Comment by osvalr** [comment]
@luistorresm @nhomar @moylop260 do you guys agree with changes in here?

File: l10n_pe_toponyms/models/res_partner.py [hunk 3]
```diff
@@ -23,27 +23,24 @@ def _get_default_country_id(self):
...
     def onchange_l10n_pe_district_id(self):
         """Fill the fields l10n_pe_province_id and state_id with the
         corresponding information extracted from the field l10n_pe_district_id
         """
-        self.update({
-            'l10n_pe_province_id': self.l10n_pe_district_id.province_id.id,
-            'state_id': self.l10n_pe_district_id.province_id.state_id.id,
-        })
+        self.l10n_pe_province_id = self.l10n_pe_district_id.province_id.id
+        self.state_id = self.l10n_pe_district_id.province_id.state_id.id
```
Comment (nhomar) [inline l10n_pe_toponyms/models/res_partner.py]: use the dictionary, do not change logic:

FYI:

This-.

```
+        self.l10n_pe_province_id = self.l10n_pe_district_id.province_id.id
+        self.state_id = self.l10n_pe_district_id.province_id.state_id.id
```

Make 2 UPDATE SQL but what it has:

```
-        self.update({
 -            'l10n_pe_province_id': self.l10n_pe_district_id.province_id.id,
 -            'state_id': self.l10n_pe_district_id.province_id.state_id.id,
 -        })
```

Makes only one.

File: l10n_pe_vat_sunat_validation/models/res_partner.py (hunk 2 above)
Comment (nhomar) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: who set the zip as required? can you show me an screenshot witht he error obtained when not set please?

Comment (osvalr) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: Sorry, it was __requested__ to be filled with de district ubigeo code

Comment (osvalr) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: @nhomar, so no error is thrown in this case

**Comment by osvalr** [comment]
@luistorresm changes you asked are done

File: l10n_pe_toponyms/models/res_partner.py [hunk 4]
```diff
@@ -23,27 +23,26 @@ def _get_default_country_id(self):
...
-    @api.one
     @api.onchange('l10n_pe_district_id')
     def onchange_l10n_pe_district_id(self):
         """Fill the fields l10n_pe_province_id and state_id with the
         corresponding information extracted from the field l10n_pe_district_id
         """
         self.update({
             'l10n_pe_province_id': self.l10n_pe_district_id.province_id.id,
-            'state_id': self.l10n_pe_district_id.province_id.state_id.id,
+            'state_id': self.l10n_pe_district_id.province_id.state_id.id
```
Comment (nhomar) [inline l10n_pe_toponyms/models/res_partner.py]: left exactly as it was (with extra *,*) try to rebase properly the change.

File: l10n_pe_vat_sunat_validation/models/res_partner.py (hunk 2 above)
Comment (nhomar) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: Can you show the specific error please?

Comment (osvalr) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: @nhomar no error is thrown, it was asked by @mmendozadelsolar to be added when the partner info is fetched

Comment (nhomar) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: can you add it afterwards to be consistent with the request in the HU and ONLY merge this as migration?

thanks.

**Comment by osvalr** [comment]
@nhomar ya puede ser mezclado esto

File: l10n_pe_vat_sunat_validation/models/res_partner.py (hunk 2 above)
Comment (osvalr) [inline l10n_pe_vat_sunat_validation/models/res_partner.py]: feature move to #47

**Pull request merged**